src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -576,6 +576,24 @@ final case class TenantApiServiceImpl(
       .recover { case _: TenantNotFound => None }
   } yield tenant.map(_.toManagement)
 
+  override def getTenantBySelfcareId(selfcareId: String)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerTenant: ToEntityMarshaller[Tenant]
+  ): Route = authorize(ADMIN_ROLE, API_ROLE, M2M_ROLE, SECURITY_ROLE, SUPPORT_ROLE) {
```

Comment:
if this route is required by the job (and to keep retrocompatibility with the management endpoint), an additional role is required
```suggestion
  ): Route = authorize(ADMIN_ROLE, API_ROLE, M2M_ROLE, SECURITY_ROLE, INTERNAL_ROLE, SUPPORT_ROLE) {
```